spec/system/admin/review_period_spec.rb

```diff
@@ -65,7 +98,7 @@
 
     within(".old-table tfoot") do
       expect(page).to have_selector("th", text: "Total")
-      expect(page).to have_selector("td", text: "$2.0")
+      expect(page).to have_selector("td", text: "$1.0")
```

Comment:
I'm curious why this only had the single decimal point before. Does the test pass if we ask for `"$1.00"`?

Reply:
Yea, it passes, and so does "$1".  Looks like `have_selector` looks for elements that _contain_ the text passed in.  You can also pass in `exact: true` if you want an exact match.